external-import/misp/src/misp.py

```diff
@@ -520,7 +516,7 @@ def process_events(self, work_id, events) -> int:
             )
             # Markings
             if "Tag" in event["Event"]:
-                event_markings = self.resolve_markings(event["Event"]["Tag"])
+                event_markings = Misp.resolve_markings(event["Event"]["Tag"])
```

Comment:
You can reference directly to the staticmethods, no need to reference to the current class itself.
```suggestion
                event_markings = resolve_markings(event["Event"]["Tag"])
```